controllers/users.js

```diff
@@ -1,48 +1,52 @@
-const { v4: uuidv4 } = require('uuid');
-const userModel = require('../db/user.model');
+const User = require('../models/user');
+const {
+    Exception
+} = require('../exceptions/Exception');
 const { 
+    UserValidationException,
     UserAlreadyExistsException, 
     UserNotFoundException 
 } = require('../exceptions/userExceptions');
+const {
+    DatabaseException
+} = require('../exceptions/databaseExceptions');
 
 class UsersController {
-    static addUser(username, password) {
-        if(userModel.find({ username }).length > 0) {
+    static async addUser(username, password) {
+        if(await User.count({ where: { username: username } })) {
```

Comment:
Just for being more readable don't use await in if, for or other structures do like this

```javascript
const resp = await User.count({ where: { username: username } })
if(resp){
   //...
}
```